scripts/functions/selector_interpreters

```diff
@@ -280,6 +280,8 @@ __rvm_truffleruby_set_rvm_ruby_url()
       linux-aarch64)  truffleruby_artifact_id="FD40BA2367C226B6E0531518000AE71A" ;;
       darwin-amd64)   truffleruby_artifact_id="FD4AB182EA51EDFDE0531518000AF13E" ;;
       darwin-aarch64) truffleruby_artifact_id="FD40BBF6750C366CE0531518000ABEAF" ;;
```

Comment:
Could you remove these two lines?
As on line 254 the platform is either `macos` or `linux`, never `darwin`

Reply:
Sure thing - updated to have "macos" only